### ISSUE TITLE
`Graph.Pack`: `parse_dot_file` numbers from 1, not from 0

### ISSUE THREAD

**Issue by eponier** [backtracking/ocamlgraph#59]
Contrary to `Rand` which starts from `0`. This implies that doing in sequence `dot_output` then `parse_dot_file` produces a graph with nodes numbered differently. The culprit is https://github.com/backtracking/ocamlgraph/blob/726e8d6e31224eb70f75655579da9a374e910b98/src/pack.ml#L149.

**Comment by backtracking** [comment]
You're right, that's inconsistent. It would be easy to fix (replacing "let new_node = ref 0" with 1 instead) but I'm a bit afraid it would break existing codes. Still hesitating...
